src/operator/contrib/multibox_prior.cc

```diff
@@ -58,7 +58,7 @@ inline void MultiBoxPriorForward(const Tensor<cpu, 2, DType> &out,
       float size = sizes[0];
       for (int j = 1; j < num_ratios; ++j) {
         float ratio = sqrtf(ratios[j]);
-        float w = size * ratio / 2;
+        float w = size * in_height / in_width * ratio / 2;
```

Comment:
@zhreshold @piiswrong Can you please provide reasoning and understanding why the term `in_height / in_width` has been multiplied here?

Since the [paper](https://arxiv.org/pdf/1512.02325.pdf) Section 2.2: Choosing scales and aspect ratios for default boxes mentions that 
`w = s * sqrt(ratio)`
`h = s / sqrt(ratio)`

Thanks in advance!